fitbenchmarking/utils/tests/test_options_minimizers.py

```diff
@@ -75,7 +75,8 @@ def test_minimizer_ralfit(self):
         """
         Checks valid ralfit minimizers are set correctly
         """
-        expected = ['gn', 'gn_reg', 'hybrid', 'hybrid_reg']
+        expected = ['gn', 'gn_reg', 'hybrid', 'hybrid_reg',
+                    'newton', 'newton_reg']
```

Comment:
Does this only look at the default minimizers? I can't tell?

Reply:
It does (I only found this out as the tests failed when I put all valid minimizers in).  I think that means the doc strings are all wrong, but that seems to be the case throughout the file -- I'll open an issue to fix that.